sensuctl/README.md

```diff
@@ -0,0 +1,9 @@
+# Habitat package: sensuctl
```

Comment:
Based on https://github.com/habitat-sh/core-plans/blob/master/README_TEMPLATE_FOR_PLANS.md can you please add the required sections for README:

- Maintainers
- Type of package
- Usage
- Bindings (if applicable)
- Topologies (if applicable)
- Update Strategies (if applicable)
- Scaling (if applicable)
- Monitoring (if applicable)